middlewares/validators/user.js

```diff
@@ -19,8 +18,7 @@ const updateUser = async (req, res, next) => {
     instagram_id: joi.string().optional(),
     site: joi.string().optional(),
     isMember: joi.boolean().optional(),
-    tokens: joi.object().optional(),
-    website: joi.string().optional()
+    tokens: joi.object().optional()
```

Comment:
@swarajpure , we won't actually be storing tokens via this route, right?
So this validation can be removed.

Reply:
Yes, that can be removed as well

Reply:
@TD-17